apps/sphinx-relay/docker-compose.yml

```diff
@@ -8,7 +8,7 @@ x-logging: &default-logging
 services:
   sphinx-relay:
     container_name: sphinx-relay
-    image: sphinxlightning/sphinx-relay:v2.0.3@sha256:9864925add480da27f04b82226d4dc0144c99941fae8c8e6e748c9705ec3bd36
+    image: sphinxlightning/sphinx-relay:v2.0.7@sha256:e20873740ad910f1417cb8327a90e10278c8ac8c3ca22b9d1e5ff9c55b232f27
```

Comment:
Where did you get this digest from?

The digest I'm getting is `54b16d19786ab215b231d1cb7dd4c3ead55fe02a437c52493086716f5d858a98`. Did you maybe pin to a hash of an individual image, and not the multi-arch manifest? If so it means all architectures will pin down the same single arch image and it'll break on either x86 or ARM.

You can just `docker pull` to get the digest

```
$ docker pull sphinxlightning/sphinx-relay:v2.0.7
v2.0.7: Pulling from sphinxlightning/sphinx-relay
Digest: sha256:54b16d19786ab215b231d1cb7dd4c3ead55fe02a437c52493086716f5d858a98
Status: Image is up to date for sphinxlightning/sphinx-relay:v2.0.7
docker.io/sphinxlightning/sphinx-relay:v2.0.7
```

Reply:
That's exactly what happened! My bad the digest is fixed now, will be more careful in the future.